src/contents/button.tsx

```diff
@@ -1,12 +1,13 @@
 import type { PlasmoCSConfig, PlasmoGetInlineAnchor } from "plasmo";
+import { Storage } from "@plasmohq/storage"
 import cssText from "data-text:../button/button.css"
 import { buttonContributions, type ButtonContributionParams, isSiteSuitable } from "../button/button-contributions";
 import { GitpodButton } from "../button/button";
 import { type ReactElement } from "react";
 import React from "react";
 
 export const config: PlasmoCSConfig = {
-  matches: ["<all_urls>"]
+  matches: ["https://*/*"]
```

Comment:
why this change?

Reply:
This should improve performance a tiny bit because we don't try to display buttons on some custom URL schemes like `urn://` or `file://`. Thinking twice about it (thanks!) reveals that the change is quite silly, because it would break http-only self-hosted installs of SCMs. I will keep the https requirement up for the Dashboard integration but revert the changes here.

Reply:
Reverted in https://github.com/gitpod-io/browser-extension/pull/92/commits/353e371de634ec51e7791351e7d6b84b77a4475e